mediathread/templates/projects/sequence_assignment_edit.html

```diff
@@ -113,7 +113,34 @@
                             </ul>
                         </nav>
                     </div>
-                    <div data-page="3" data-page-content="due-date" class="hidden page">
+                    <div data-page="3" data-page-content="custom-instructions" class="hidden page">
+                        <h3>2. Write custom instructions</h3>
+                        <div class="form-group">
+                            <label>Primary Video Instructions</label>
+                            <div>Offer directed instructions on the primary video a student should use:</div>
+                            <ul>
+                                <li>@todo</li>
+                                <li>140 character limit</li>
+                            </ul>
+                            <textarea name="custom_instructions_1" class="form-control mceEditor">{{form.instance.custom_instructions_2}}</textarea>
```

Comment:
These two textareas could have `maxlength="140"`, since this field is limited to 140 characters.